components/draggable/style.scss

```diff
@@ -0,0 +1,19 @@
+body.dragging {
```

Comment:
Is a `.dragging` class unique enough to avoid potential conflicts?

Reply:
Definitely not - need to think of a better name here.

Reply:
Updated the className to use `is-dragging-components-draggable` Let me know if you think of a better name.